src/dmd/traits.d

```diff
@@ -139,10 +139,11 @@ shared static this()
         "getVirtualIndex",
         "getPointerBitmap",
         "isZeroInit",
-        "getTargetInfo"
+        "getTargetInfo",
+        "getLocation"
```

Comment:
Always using trailing commas would have made the diff one line shorter.

Reply:
Done.